examples/http-server/README.md

```diff
@@ -4,17 +4,13 @@ This GoFr example demonstrates a simple HTTP server which supports Redis and MyS
 
 ### To run the example follow the steps below:
 
-- Run the docker image of Redis
+- Run the docker image of the application
 ```console
-docker run --name gofr-redis -p 2002:6379 -d redis:7.0.5
+docker-compose up -d
 ```
 
-- Run the docker image of MySQL
-```console
-docker run --name gofr-mysql -e MYSQL_ROOT_PASSWORD=password -e MYSQL_DATABASE=test -p 2001:3306 -d mysql:8.0.30
-```
+To test the example, follow these steps:
+
+1. Open your browser and navigate to `http://localhost:9000/hello`.
+2. To view the trace in Jaeger, open `http://localhost:16686`.
```

Comment:
now it won't be visible in jaeger we can update the link in message to view on the gofr tracer website

https://tracer.gofr.dev